### DISCUSSION TITLE
checkbox to indicate if device needs to have maintenance tasks planned.

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#66] [Ideas]
I would like a checkbox (default checked) for an inventory item to indicate if preventive maintenance is required. We have some items that we don't want to do preventive maintenance on. So no task is planned for these devices. When we filter the device list on empty Last event and empty Next event to see if any device does not have maintenance task planned yet (Which might happen if next task is forgotten to plan when finishing the current task, or when adding a new inventory item and forgetting to plan). But if we filter on empty Last and Next event then all these items that do not need maintenance also show up. So a column to filter any device that does not need maintenance will be very helpful. Similar to the 'is medical device' checkbox/column.

**Comment by YvesR** [comment]
I don't think the idea of maintaining extra data is ideal, but a virtual column with a yes/no if at least one service interval is stored, would this map to the UseCase?
